src/test/java/com/atlassian/bitbucket/jenkins/internal/fixture/ProjectType.java

```diff
@@ -0,0 +1,52 @@
+package com.atlassian.bitbucket.jenkins.internal.fixture;
+
+import hudson.model.FreeStyleProject;
+import hudson.model.Job;
+import hudson.model.TopLevelItem;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+
+import java.util.EnumSet;
+import java.util.Optional;
+
+/**
+ * Describes the kinds of jobs that the {@link JenkinsProjectRule} can create and run tests against. To support another
+ * job type, it should be added to this list, and the project rule should be updated.
+ */
+public enum ProjectType {
+    FREESTYLE(FreeStyleProject.class),
+    PIPELINE(WorkflowJob.class);
```

Comment:
My knowledge of Jenkins Job types is still a bit sketchy, but does multi-branch build fall under wither of these categories?